lib/templates/rails/credentials/credentials.yml.tt

```diff
@@ -0,0 +1,10 @@
+# Used as the base secret for all MessageVerifiers in Rails, including the one protecting cookies.
+secret_key_base: <%= secret_key_base %>
+
+active_record_encryption:
+  primary_key: <%= SecureRandom.alphanumeric(32) %>
+  deterministic_key: <%= SecureRandom.alphanumeric(32) %>
+  key_derivation_salt: <%= SecureRandom.alphanumeric(32) %>
+
+# database:
+#   password: <%= SecureRandom.alphanumeric(32) %>
```

Comment:
I see that this template is used when following the instructions to execute `bin/rails credentials:edit --environment production`. But the next instruction is to uncomment these two lines. Should we instead just uncomment them within this template file?